src/Mvc/Mvc.Formatters.Xml/test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -242,19 +231,6 @@ public async Task BuffersRequestBody_ByDefaultUsingMvcOptions()
 
             Assert.Equal(expectedInt, model.SampleInt);
             Assert.Equal(expectedString, model.sampleString);
-
-            Assert.True(httpContext.Request.Body.CanSeek);
-            httpContext.Request.Body.Seek(0L, SeekOrigin.Begin);
-
-            result = await formatter.ReadAsync(context);
```

Comment:
The guidance for users should be to explicitly call `EnableRewind` in a middleware prior to getting to MVC if they want to read the body again after input formatting rather than rely on the formatter.